.circleci/config.yml

```diff
@@ -273,7 +273,7 @@ commands:
           name: ng test
           command: |
             mkdir -p /tmp/junit
-            ng test ddp-sdk --watch=false --browsers ChromeHeadlessNoSandbox
+            ng test ddp-sdk --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
```

Comment:
Explicitly saying which reporter to use now that we have multiple ones. More about this below.